RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -12,8 +12,8 @@
 # run only if there are high pT jets
 jetsForCoreTracking = cms.EDFilter('CandPtrSelector', src = cms.InputTag('ak4CaloJetsForTrk'), cut = cms.string('pt > 100 && abs(eta) < 2.5'), filter = cms.bool(False))
 
-jetsForCoreTrackingBarrel = jetsForCoreTracking.clone( cut = 'pt > 100 && abs(eta) < 2.5' )
-jetsForCoreTrackingEndcap = jetsForCoreTracking.clone( cut = 'pt > 100 && abs(eta) > 1.4 && abs(eta) < 2.5' )
+jetsForCoreTrackingBarrel = jetsForCoreTracking.clone( cut = 'pt > 100 && abs(eta) < 1.4' )
+jetsForCoreTrackingEndcap = jetsForCoreTracking.clone( cut = 'pt > 100 && abs(eta) < 2.5' )
```

Comment:
Please forgive my ignorance, but this collection is now exactly equal to above one, jetsForCoreTracking? I.e. we are running it twice, shouldn't stay with a pure endcap selection, e.g. abs(eta) > 1.4 && abs(eta) < 2.5'  ?

Reply:
> Please forgive my ignorance, but this collection is now exactly equal to above one, jetsForCoreTracking? I.e. we are running it twice, shouldn't it stay with a pure endcap selection, e.g. abs(eta) > 1.4 && abs(eta) < 2.5' ?

this is intentional. The Barrel/Endcap variants run only with `seedingDeepCore` and in that case `jetsForCoreTracking` does not run.
The naming may be confusing and could instead be made more explicit: `Barrel`->`DeepCore`, `Endcap` -> `Recovery` or something like that